OnionBrowser/Browsing/BrowsingViewController+Tabs.swift

```diff
@@ -72,8 +72,8 @@ UICollectionViewDropDelegate, TabCellDelegate {
 				let tab = tabs[indexPath.row]
 
 				cell.title.text = tab.title
-
-				tab.add(to: cell.container)
+				
+				cell.preview.image = tab.snapshot
 				tab.isHidden = false
 				tab.isUserInteractionEnabled = false
```

Comment:
Do we still need this, if we switch to screenshots?

Reply:
remove not needed changing of isUserInteractionEnabled